tests/foreman/api/test_rhcloud_inventory.py

```diff
@@ -103,8 +103,13 @@ def test_rhcloud_inventory_api_e2e(
     # Assert Hostnames, IP addresses, and installed packages are present in report.
     json_data = get_report_data(local_report_path)
     json_meta_data = get_report_metadata(local_report_path)
-    package_version = rhcloud_sat_host.run('rpm -qa --qf "%{VERSION}" tfm-rubygem-foreman_rh_cloud')
-
+    os_version = rhcloud_sat_host.os_version.major
+    if os_version == 8:
+        package_version = rhcloud_sat_host.run('rpm -qa --qf "%{VERSION}" rubygem-foreman_rh_cloud')
+    else:
+        package_version = rhcloud_sat_host.run(
+            'rpm -qa --qf "%{VERSION}" tfm-rubygem-foreman_rh_cloud'
+        )
```

Comment:
What do you say about the following?

```suggestion
    rhcloud_package_n = 'rubygem-foreman_rh_cloud'
    if rhcloud_sat_host.os_version.major < 8:
        rhcloud_package_n = 'tfm-' + rhcloud_package_n
    package_version = rhcloud_sat_host.run('rpm -qa --qf "%{VERSION}" rubygem-foreman_rh_cloud'
```

Reply:
Or even better:
```python
prefix='tfm-' if rhcloud_sat_host.os_version.major < 8 else '' 
package_version = rhcloud_sat_host.run(f'rpm -qa --qf "%{{VERSION}}" {prefix}rubygem-foreman_rh_cloud')
```

Reply:
@ogajduse you probably forgot to update the last line (it also misses the closing parenthesis):
```
package_version =  rhcloud_sat_host.run(f'rpm -qa --qf "%{{VERSION}}" {rhcloud_package_n}')
```

Reply:
> @ogajduse you probably forgot to update the last line (it also misses the closing parenthesis):
> 
> ```
> package_version =  rhcloud_sat_host.run(f'rpm -qa --qf "%{{VERSION}}" {rhcloud_package_n}')
> ```

Yup, that is a mistake. Fixed. However I prefer @lpramuk's code over mine.